docs/reference/fields.txt

```diff
@@ -1396,6 +1396,42 @@ You can get and set all the translations at once by using the corresponding ``_t
      { "en" => "Marvelous!", "de" => "Wunderbar!" }
 
 
+.. _present-fields:
+
+Present Fields
```

Comment:
"Present Fields" as a header doesn't really sound clear to me as a reader. How about something like: "Auto-Remove Blank Translations"